src/main/kotlin/com/chattriggers/bot/messages/learnJs.kt

```diff
@@ -0,0 +1,29 @@
+package com.chattriggers.bot.messages
+
+import com.chattriggers.bot.CTBot
+import com.jessecorbett.diskord.api.rest.client.ChannelClient
+import com.jessecorbett.diskord.dsl.footer
+import com.jessecorbett.diskord.dsl.field
+import com.jessecorbett.diskord.util.sendMessage
+import io.ktor.util.*
+import java.time.ZoneOffset
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+
+@KtorExperimentalAPI
+suspend fun ChannelClient.learnJsMessage(username: String) {
+    sendMessage("") {
+        title = "JavaScript Links"
+
+        description = """
+            You are expected to understand the basics of JavaScript before asking for help in the ChatTriggers discord. Here are some good places to start!
```

Comment:
Sounds a bit harsh, how about this:
```suggestion
            You need some basic JavaScript knowledge to use ChatTriggers, and the best place to do that is a dedicated JavaScript tutorial. Here are some good places to start!
```